content/concept-history-sharding.md

```diff
@@ -0,0 +1,57 @@
+# History Sharding
+
+Rippled sharding is a feature that manages storing and sharing of distributed historical data, as shards. As servers operate, they naturally produce a database containing the nodes from the ledgers they witnessed or acquired during network runtime.
```

Comment:
Consider re-ordering the content here slightly. It might help motivate the existence of shards before having to define them:


> As servers operate, they naturally produce a database containing the data for the ledgers they witnessed or acquired during network runtime. This ledger data is stored in the node store, but the online delete logic rotates these databases when their size exceeds configured space limitations. 

> Rippled sharding is a feature that manages storing and sharing of distributed historical data as shards, which individually contain data for a segment of ledger history. In this way, Rippled sharding provides a middle ground between running a rippled server with full-history and one with only recent ledger data. History shards do not replace the node store, but implement a reliable path towards distributed ledger history across the rippled network.